src/index.js

```diff
@@ -96,17 +99,50 @@ yargs
 	.check((argv) => {
 		return langCheck(argv.l);
 	});
-//Call convertToHtml
-try {
-	convertToHtml(
-		yargs.argv.i,
-		yargs.argv.s,
-		yargs.argv.o,
-		isFile(yargs.argv.i),
-		yargs.argv.l,
-	);
-} catch (e) {
-	console.error(`\n\n${chalk.red.bold('Error:')} ${chalk.red(e)}`);
-	process.exit(1);
+
+//configuration option
+yargs
+	.option('c', {
+		alias: 'config',
+		describe: 'Folder/File configuration JSON file location',
+		type: 'string',
+		nargs: 1
+	})
+	.check((argv) => {
+		return configurationCheck(argv.c);
+	});
+
+//if the ption is c or config, run applyConfig instead of below
+if(yargs.argv.c){
+	try {
+		const config = readConfig(yargs.argv.c);
+		convertToHtml(
+			config.input,
+			congig.stylesheet,
+			config.output,
```

Comment:
If output doesn't exist set a default value to ./dist `config.output || './dist'`